polymetis/polymetis/python/polymetis/robot_interface.py

```diff
@@ -744,3 +744,9 @@ def move_ee_xyz(
             "The method 'move_ee_xyz' is deprecated, use 'move_to_ee_pose' with 'delta=True' instead."
         )
         return self.move_to_ee_pose(position=displacement, delta=True, **kwargs)
+
+    def set_sim_state(self, robot_state: RobotState):
+        self.grpc_connection.SetSimRobotState(robot_state)
```

Comment:
You should not be needing the `GetSimRobotState` & `SetSimRobotState` services anymore, as you are able to directly call `mirror_sim_client.set_robot_state` from the User Client (a.k.a. `mirror_sim.py`). Also, these two services don't exist as this branch is not based off of the branch in which you implemented these services.